Makefile

```diff
@@ -28,7 +28,7 @@ lint:
 	@echo "--- rubocop ---"
 	bundle exec rubocop --parallel
 	@echo "--- brakeman ---"
-	bundle exec brakeman
+	bundle exec brakeman --skip-files app/services/doc_auth_router.rb
```

Comment:
Interesting: https://github.com/presidentbeef/brakeman/issues/1559

Looks like the necessary work is done in the ruby_parser dependency? https://github.com/seattlerb/ruby_parser/issues/311

Is it possible for us to pin a transitive dependency to a newer version?

Reply:
https://github.com/presidentbeef/brakeman/blob/69101ca16549319e840fc464d49d545acd5bb3e9/gem_common.rb#L12

Looks like they pin to "~>3.13" so I think we could only pin to `3.13.X` otherwise rubygems/bundler will not allow it

Reply:
interestingly, we don't have `ruby_parser` in our Gemfile.lock at all, so Brakeman must have some odd ways of downloading and resolving its dependencies